docs/3.0/deploy/infrastructure-concepts/deploy-ci-cd.mdx

```diff
@@ -316,13 +316,13 @@ so Python packages do not have to be downloaded on repeat workflow runs.
 - name: Setup Python
   uses: actions/setup-python@v5
   with:
-    python-version: "3.11"
+    python-version: "3.12"
     cache: "pip"
 ```
 
 ```
 Using cached prefect-3.0.0-py3-none-any.whl (2.9 MB)
-Using cached prefect_aws-0.4.18-py3-none-any.whl (61 kB)
+Using cached prefect_aws-0.5.0-py3-none-any.whl (61 kB)
 ```
```

Comment:
I'd vote to remove this output code block since it's output of code that isn't a part of Prefect.

Reply:
done